xml/System.Globalization/CompareInfo.xml

```diff
@@ -246,49 +246,49 @@
         <param name="string2">The second string to compare.</param>
         <param name="options">A value that defines how <paramref name="string1" /> and <paramref name="string2" /> should be compared. <paramref name="options" /> is either the enumeration value <see cref="F:System.Globalization.CompareOptions.Ordinal" />, or a bitwise combination of one or more of the following values: <see cref="F:System.Globalization.CompareOptions.IgnoreCase" />, <see cref="F:System.Globalization.CompareOptions.IgnoreSymbols" />, <see cref="F:System.Globalization.CompareOptions.IgnoreNonSpace" />, <see cref="F:System.Globalization.CompareOptions.IgnoreWidth" />, <see cref="F:System.Globalization.CompareOptions.IgnoreKanaType" />, and <see cref="F:System.Globalization.CompareOptions.StringSort" />.</param>
         <summary>Compares two strings using the specified <see cref="T:System.Globalization.CompareOptions" /> value.</summary>
-        <returns>A 32-bit signed integer indicating the lexical relationship between the two comparands.  
-  
- <list type="table"><listheader><term> Value  
-  
- </term><description> Condition  
-  
- </description></listheader><item><term> zero  
-  
- </term><description> The two strings are equal.  
-  
- </description></item><item><term> less than zero  
-  
- </term><description><paramref name="string1" /> is less than <paramref name="string2" />.  
-  
- </description></item><item><term> greater than zero  
-  
- </term><description><paramref name="string1" /> is greater than <paramref name="string2" />.  
-  
+        <returns>A 32-bit signed integer indicating the lexical relationship between the two comparands.
```

Comment:
The `<returns>` value is too large. I think it's expected to move the list to the remarks. Would you agree, @mairaw / @rpetrusha ?

If the list is moved to remarks, it needs to be added with markdown. It could be added as a simple list (with `*` prefix) or you could add it as a table. If you opt for table, this page has been really useful to quickly generate it: https://www.tablesgenerator.com/markdown_tables

Reply:
Same comment would apply for the other similar lists in the rest of the review.

Reply:
@carlossanlop  this is unrelated to the changes I am doing. I suggest you try to fix this in other PR. Has this worked before?

Reply:
Okay, it can be addressed in another PR.